notebooks/featurestore/hsfs/time_travel/time_travel_python.ipynb

```diff
@@ -1315,7 +1315,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
    "language": "python",
    "name": "python3"
```

Comment:
This is a PySpark notebook, even though it's called `time_travel_python` it's just to distinguish it from the Scala version of the notebook.

Reply:
Fixed